README.md

```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
 
 The segmentation workflow consists of the following steps each defined by a "rule". 
 The output of each rule is store to specific folder that is produced by the worrkflow:
```

Comment:
```suggestion
The output of each rule is store to specific folder that is produced by the workflow:
```

Reply:
@johnbradley I added two more small corrections. Can you double check?